Automattic/wp-calypso
# Plans: add plan-feature-items

This is part of the plans redesign, which breaks out the plan features item component from @adambbecker's work in #5715
![b6a11926-2cd1-11e6-8322-f88c322fbdcd](https://cloud.githubusercontent.com/assets/1270189/15950642/7de29322-2e65-11e6-8040-666615b10657.png)

This PR adds the plans feature items to [devdocs](http://calypso.localhost:3000/devdocs/app-components/plan-features)
<img width="463" alt="screen shot 2016-06-09 at 5 14 25 pm" src="https://cloud.githubusercontent.com/assets/1270189/15950658/a3d7c606-2e65-11e6-8790-524335d621af.png">

## Testing Instructions
- No functional changes to Calypso.
- Navigate to http://calypso.localhost:3000/devdocs/app-components/plan
- You should see the PlanFeatures component as shown above

cc @lamosty @rralian @mtias @rodrigoi
